src/slack/slack.js

```diff
@@ -832,9 +832,9 @@ export class Slack {
   }
 
   async removeReaction(name, channel, timestamp) {
-    const {ok, error} = await this.apiCall('reactions.remove', {name, channel, timestamp}, {passError: true})
-
-    if (ok === false && error !== 'no_reaction') {
+    try {
+      await this.boltApp.client.reactions.remove({name, channel, timestamp})
```

Comment:
you can remove the `apiCall` function and also other functions it used, so `makeApiCall` is also not needed now

Reply:
we need `makeApiCall` for `logError` function now.